### PR TITLE
First PR from roger926.rodriguez@gmail.com

### PR DESCRIPTION
**Pull request by TrustedSeven**
Hi
I made PR for wing order change(which was incorrect) and rotation in one direction.
If you have questions email me: roger926.rodriguez@gmail.com

### CODE REVIEW

File: components/stlViewer/index.tsx [hunk 1]
```diff
@@ -55,23 +55,28 @@ export default function StlViewer({
...
+						z = 0;
+						console.log("xxxxxxxxxxxx");						
 					} else if (pos.y > pos.x && pos.y > pos.z) {
-						y = 0.1;
-						x = pos.y / (pos.y * 10)
-						z = pos.z / (pos.y * 10)
+						y = 0.2;
+						x = 0;
+						z = 0;
+						console.log("yyyyyyyyyyyy")
```
Comment (SashaM-Solicy) [inline components/stlViewer/index.tsx:71]: remove this console log

File: components/stlViewer/index.tsx [hunk 2]
```diff
@@ -55,23 +55,28 @@ export default function StlViewer({
...
 					let y = 0;
 					let z = 0;
 					if (pos.x > pos.y && pos.x > pos.z) {
-						x = 0.1;
-						y = pos.y / (pos.x * 10)
-						z = pos.z / (pos.x * 10)
+						x = 0;
+						y = 0.2;
+						z = 0;
+						console.log("xxxxxxxxxxxx");
```
Comment (SashaM-Solicy) [inline components/stlViewer/index.tsx:66]: remove logs

**Review by SashaM-Solicy — COMMENTED** [review]
Also please attach how it's working for now

File: components/stlViewer/index.tsx (hunk 1 above)
Comment (TrustedSeven) [reply components/stlViewer/index.tsx:71]: each conditional statement determines the direction of the rotation based on camera view.

File: components/stlViewer/index.tsx (hunk 2 above)
Comment (TrustedSeven) [reply components/stlViewer/index.tsx:66]: sure

**Comment by TrustedSeven** [comment]
1. order of wings were not correct
wing no.1 and wing no.3 were swapped.
I made changes to them and swapped them again

2. tectonic rotation not working well
tectonic rotate direction was not correct
i clarified the direction for each camera direction(x,y,z) and set rotate direction

This PR contains above changes

**Pull request closed without merging**